rps.js

```diff
@@ -0,0 +1,22 @@
+const playGame =function (user_choice){ 
+    let choices=['rock','paper','scissors'];   
+    let computer_choice =Math.floor(Math.random() * choices.length) - 1;
+    if (computer_choice===user_choice){
+        console.log('Computer picked: '+ computer_choice+'. User picked: '+user_choice+'. User tie');
+    }else{
+            if(computer_choice===choices[0]&&user_choice===choices[1]){

```

Comment:
the game doesn't play because the computer is not accessing the the choice array, a simple way to do this 

would be to do `choices[computer_choice]` in the place of computer_choice this will make your code work